### PR TITLE
wrap collection summary in helpers.format_record_html

### LINKED ISSUE #3795: Line break and ampersand work in Collection description needs to extend to "Context" box in item record view
Related to: https://github.com/sul-dlss/SearchWorks/issues/3699

I just noticed that we have encodings (i.e. &#13) and issues with line breaks in the Collection description that appears in the "Context" box on the item record view.

Example record: https://searchworks.stanford.edu/view/sy691hy9213
image

### PR DESCRIPTION
**Pull request by dnoneill**
closes #3795

### CODE REVIEW

**Review by jcoyne — CHANGES_REQUESTED** [review]
Would you mind adding a component unit test for this?

File: spec/components/access_panels/in_collection_component_spec.rb [hunk 1]
```diff
@@ -16,4 +16,20 @@
       expect(page).to have_no_content("Digital content")
     end
   end
+
+  describe 'merged records with collection members' do
```
Comment (jcoyne) [inline spec/components/access_panels/in_collection_component_spec.rb:20]: would you mind updating this to reflect that we're testing escaped characters in the summary field?

Comment (dnoneill) [reply spec/components/access_panels/in_collection_component_spec.rb]: not at all. good catch.

File: spec/components/access_panels/in_collection_component_spec.rb [hunk 2]
```diff
@@ -16,4 +16,20 @@
       expect(page).to have_no_content("Digital content")
     end
   end
+
+  describe 'merged records with collection members that have encoded summaries' do
```
Comment (jcoyne) [inline spec/components/access_panels/in_collection_component_spec.rb:20]: This test is for the AccessPanels::InCollectionComponent, but the change you made was to the AccessPanels::CollectionComponent. Can you create a test file for that component and move your test there?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: spec/components/access_panels/in_collection_component_spec.rb (hunk 2 above)
Comment (jcoyne) [reply spec/components/access_panels/in_collection_component_spec.rb:20]: Actually, it appears that these two components wrap one another, so we can keep it like this.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**